pages/about/index.tsx

```diff
@@ -131,6 +132,71 @@ const About = () => (
               </div>
             </div>
 
+            <div className="col-span-12 md:col-span-6">
+              <h2 className="h3 mb-6">Mastodon, Inc.</h2>
+
+              <p className="b1 mb-4">
+                Mastodon, Inc. (EIN 92-3333630) is a 501(c)(3) non-profit entity in the United States
+                that supports the growth and operational capabilities of Mastodon, including being able
+                to receive tax-deductible U.S. donations and in-kind support.
+              </p>
+            </div>
+
+            <div className="col-span-12 md:col-span-6">
+              <a id="board" className="invisible block relative -top-32"/>
```

Comment:
Calling out the anchor link to the board (`/about#board`). Can change this to something.

Reply:
I would prefer `us-board`